tests/influence/test_torch_differentiable.py

```diff
@@ -10,31 +10,29 @@
 import numpy as np
 import pytest
 
-from pydvl.utils import (
-    linear_regression_analytical_derivative_d2_theta,
-    linear_regression_analytical_derivative_d_theta,
-    linear_regression_analytical_derivative_d_x_d_theta,
+from .conftest import (
+    linear_derivative_analytical,
+    linear_hessian_analytical,
+    linear_mixed_second_derivative_analytical,
+    linear_model,
 )
 
 try:
+    import torch
     import torch.nn.functional as F
+    from torch import nn
 
     from pydvl.influence.frameworks import TorchTwiceDifferentiable
-    from pydvl.influence.model_wrappers import TorchLinearRegression
 except ImportError:
     pass
```

Comment:
We can replace this try except block with:

```suggestion
torch = pytest.importorskip("torch")

import torch.nn.functional as F
from pydvl.influence.frameworks import TorchTwiceDifferentiable
from pydvl.influence.model_wrappers import TorchLinearRegression
```

This is will skip this test file if pytorch cannot be imported.